README.md

```diff
@@ -1,2 +1,180 @@
-# ReactCapstoneProject
-This React capstone project (remember what are they?)is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+![](https://img.shields.io/badge/ReacrReduxCapstone-orange)
+
+# Covid 19 Africa metric data.
+
+## Description
+- This React capstone project (remember what are they?)is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+  
+
+||     Home       ||        Details        ||
+|-|----------------------|-| ----------------------|-|
+||![screenshot](./src/images/screenshot3.png)||![screenshot](./src/images/screenshot4.png)||
+
+## Learning objectives
+
+- Use React documentation.
+- Use React components.
+- Use React props.
+- Use React Router.
+- Connect React and Redux.
+- Handle events in a React app.
+- Write integration tests with a React testing library.
+- Use styles in a React app.
+- Use React life cycle methods.
+- Apply React best practices and language style guides in code.
+- Use store, actions and reducers in React.
+
+
+### Built With
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+This project is build with:
+
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
+- ![HTML5](https://img.shields.io/badge/-HTML5-000000?style=flat&logo=html5&logoColor=ffffff&labelColor=E34F26) ![CSS3](https://img.shields.io/badge/-CSS3-000000?style=flat&logo=css3&logoColor=ffffff&labelColor=1572B6)
+- ![React](https://img.shields.io/badge/-React-000000?style=flat&logo=react)  ![Redux](https://img.shields.io/badge/-Redux-000000?style=flat&logo=redux&logoColor=ffffff&labelColor=violet)
+
+## Live Demo Link
+
+### [Live Demo](https://metricdatacovid19.netlify.app/)
+
+## Loom Video Link
+
+### [Loom](https://www.loom.com/share/cdba46e67d90463a8dcfe7858b2cd7b5)
```

Comment:
- [ ] Your video has no audio. Kindly redo the video with sound presenting the project and remember to show the piece of code you find interesting or are proud of making.